server/api/mentions_handler.py

```diff
@@ -0,0 +1,40 @@
+import os
+from datetime import datetime
+from pymongo import MongoClient
+from bson import json_util
+import json
+from flask import Blueprint, request, jsonify
+mentions_handler = Blueprint('mentions_handler', __name__)
+
+
+@mentions_handler.route('/mentions', methods=['GET', 'POST', 'DELETE'])
+def mentions():
+    connection_string = 'mongodb+srv://{username}:{password}@{server}/?retryWrites=true&w=majority'
+    client = MongoClient(connection_string.format(
+        username=os.getenv('USERNAME'),
+        password=os.getenv('PASSWORD'),
+        server=os.getenv('DB_SERVER')
+    ))
+    db = client[os.getenv('DB_NAME')]
+    mentions_collection = db.mentions
+    # this POST functionality is just to load some data for testing
+    if request.method == 'POST':
+        result = mentions_collection.insert_one({
+            'title': 'Foo',
+            'content': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+            'platform': 'Twitter',
+            'image': 'https://via.placeholder.com/150',
+            'date': datetime.now(),
+            'popularity': 1
+        })
+        return str(result.inserted_id)
+    elif request.method == 'DELETE':
+        result = mentions_collection.delete_many({})
+        return jsonify({'documentsDeleted': result.deleted_count})
```

Comment:
am I mistaken? Doesn't flask automatically jsonify all responses?

Reply:
change log for 1.1.0:
```
Allow returning a dictionary from a view function. Similar to how returning a string will produce a text/html response, returning a dict will call jsonify to produce a application/json response. :pr:`3111`
```

Reply:
Oh that's cool! I will update accordingly 👍